src/components/Clue/Clue.tsx

```diff
@@ -54,9 +57,12 @@ function Clue({
       tabIndex={0}
     >
       <span className="Clue__num">{num}</span>
-      <span className="Clue__text" data-text={text}>
-        {text}
-      </span>
+      <span
+        className="Clue__text"
+        // eslint-disable-next-line react/no-danger
+        dangerouslySetInnerHTML={{ __html: sanitizedText }}
+        data-text={sanitizedText}
```

Comment:
I think it makes more sense to use `decodeHtmlEntities(stripHtml(text))` as that will be more representative of the HTML string length.